### ISSUE TITLE
Will this project migrate from 2.x to 3.0?

### ISSUE THREAD

**Issue by mpjraaij** [walkor/phpsocket.io#262]
Currently the project runs on 2.x. Do you plan on upgrading to 3.0?

**Comment by walkor** [comment]
I'm very sorry for the limited energy. There is no plan at present. I would be very grateful if anyone could do it.

**Comment by mader89** [comment]
v2版本太老了，能否考虑更新一下

**Comment by marcosmarcolin** [comment]
@walkor Hello, I'm going through the same problem, not being able to make it work with version >= 3, is that it? Does it only work on versions <= 3?

**Comment by walkor** [comment]
Yes, it only work on versions < 3. There is no plan on upgrading.

**Comment by wilianmaique** [comment]
too bad, that would be great, I don't understand why the socket.io people don't keep a php repository

**Comment by marcosmarcolin** [comment]
Closed due to inactivity or open for too long.

If you still need support, please open a new issue.
